app/views/users/index.html.erb

```diff
@@ -1,8 +1,15 @@
-<h1>List of all static users:</h1>
-<ul>
-  <li>User 1</li>
-  <li>User 2</li>
-  <li>User 3</li>
-  <li>User 4</li>
-  <li>User 5</li>
-</ul>
\ No newline at end of file
+<div class="user-container">
+  <% @users.each do |user| %>
+    <div class="user-wrapper">
+      <div class="photo-box">
+        <img class="user-img" src="
+        <% user.photo %>" alt="<%= user.name %>">
+      </div>
+
+      <div class="text-box">
+        <h3><%= user.name %></h3>
+        <p>Number of posts: <%= user.posts_counter %></p>
```

Comment:
- [ ] Great job implementing some required changes from the previous reviewer. However, there's one major required change you're yet to implement. `Clicking on each user should take us to the user show page`. It is a click handler. Look at this note from the project requirements which says :point_down: 

## Note: Analyze the wireframes deeply and try to identify all the necessary elements and how a user would navigate from one page to another.

You don't expect your user to be manually navigating from one page to another. That would not give a good user experience. From the pages below that note, it is clearly shown that clicking on a user takes you to the user's show page. Take a look and implement the changes accordingly.

### Project Requirement

![user-show-page](https://github.com/M-Anwar-Hussaini/BlogApp/assets/26597432/5152f0a7-1aca-4a32-a26f-189cf8515d7e)